src/handler_error.rs

```diff
@@ -206,12 +206,10 @@ pub async fn handle_rejection(rej: Rejection) -> Result<impl Reply, Rejection> {
                     ))),
                     StatusCode::NOT_FOUND,
                 ),
-                ApiError::JsonEncodingFailed(field_name, err) => reply::with_status(
-                    reply::json(&internal_json_encoding_failed_error(
-                        Some(field_name.clone()),
-                        Some(err.to_string()),
-                    )),
-                    StatusCode::BAD_REQUEST,
+                // We explicitly ignore the error message as it should not be passed to the user
```

Comment:
Why? How are we going to figure out what's wrong without the error message?

Reply:
Aleš said not to pass the Error to the user. The argument was that we shouldn't be bothering clients with these errors. These errors really shouldn't be happening anyway, so it's not a big issue.

Reply:
As I understand it, the current instances of `InternalServerError`s cannot leak sensitive information but generally an internal server error could leak sensitive information so therefore we do not log this to the user.

It's done as per Aleš' wishes, I'll merge.